Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2395,6 +2394,10 @@ extension BrowserViewController: TabDelegate {
       tab.requestBlockingContentHelper,
     ]
     
+    if let braveSkusHandler = BraveSkusScriptHandler(tab: tab) {
+      injectedScripts.append(braveSkusHandler)
```

Comment:
No changes here but SKUServiceFactory may return optional so had to unwrap it first